Formula/now-cli.rb

```diff
@@ -0,0 +1,42 @@
+require "language/node"
+
+class NowCli < Formula
+  desc "The command-line interface for Now"
+  homepage "https://zeit.co/now"
+  url "https://github.com/zeit/now-cli/archive/15.8.0.tar.gz"
+  sha256 "a0ac758dfe87e67cbfded18d44d8eca657f4ec1767f44ee3b4380738859e836c"
+
+  depends_on "node" => :build
+
+  def install
+    inreplace "package.json" do |s|
+      s.gsub! /^.*"postinstall".*$/, "" # don't run postinstall
+    end
+
+    system "npm", "install", *Language::Node.local_npm_install_args
+    system "npm", "run", "build"
+
+    # Read the target node version from package.json
+    target = IO.read("package.json").match(/\"(node\d+(\.\d+){2})-[^"]+\"/)[1]
+
+    # This packages now-cli together with a patched version of node
+    pkg_args = %W[
+      --c=package.json
+      --o=now
+      --options=no-warnings
+      --targets=#{target}
+    ]
```

Comment:
Is this able to use our `node@10` (or even `node`) at runtime?

Alternatively could you please add at least the `--build` option here. It's preferred to not pull large prebuilt binaries (such as custom prebuilt node binaries here) during the formula build and instead build everything from source.

Reply:
https://github.com/zeit/pkg/issues/326
It is requested, but seems not supported yet.
`--build` makes sense to me, I will add it soon.

Reply:
Adding `--build` makes sense here, because unfortunately using our `node` at runtime would be overly complicated.

Also don't forget to replace `assert File.exist?("bash/index.sh")` `assert_predicate "bash/index.sh", :exist?` and so on in `test` to fix the audit error.

Reply:
@chrmoritz Sorry but I have trouble with `--build`, see https://github.com/zeit/pkg/issues/739 for details, would you like to help?

Reply:
I can reproduce this build failure locally, but is must have something to do with their build config/patches, because we don't see this for official node builds.

An alternativ would be trying to do something like this https://github.com/chrmoritz/homebrew-core/commit/4521080efdd15534fe8d4685823e565cb01fc545 to get it to work against our `node` at runtime.

But regardless which method we choose, we should patch the update check. Letting it suggest to update it via npm every time a new version is available, even when it wasn't installed that way is probably a bad idea.

Reply:
@chrmoritz Your solution is great! I've updated the file with tiny improvements, please help to review it again.